examples/kubernetes_orchestration/README.md

```diff
@@ -126,10 +139,25 @@ zenml stack register k8s_stack
     -a s3_store 
     -o k8s_orchestrator 
     -c ecr_registry
+```
+
+Let's set this stack as active, so we use it by default for the remainder of
+this example:
+
+```bash
 zenml stack set k8s_stack
 ```
 
-Next, call `zenml stack up` to provision and start the metadata store.
+Next, provision and start the metadata store with the following command:
+
+```bash
+zenml stack up
+```
+
+If everything went well, you should see logs messages similar to the following
+in your terminal:
+
+![zenml stack up output](assets/zenml_stack_up_output.png)
 
 ## :computer: Run Pipeline Locally
```

Comment:
This heading might be a bit confusing to the reader? Didn't we just set up a Kubernetes cluster in the cloud, why is this running locally.

Reply:
Good point, I added a bit more explanation what's going on there.

Reply:
I would even change the heading title to "Run the pipeline"

Reply:
Ah, yeah makes sense!